src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -3097,7 +3097,7 @@ private static function mkMatrix(...$args)
      * @param string $formula
      * @param null|\PhpOffice\PhpSpreadsheet\Cell\Cell $pCell
      *
-     * @return bool
+     * @return array
```

Comment:
This is should allow at least `array` and `false`. And maybe `void` as well, which is unfortunately a bit incorrect, so it might be best to leave it out.